packages/linter/src/utils/convert-tslint-to-eslint/__snapshots__/project-converter.spec.ts.snap

```diff
@@ -94,6 +72,28 @@ Object {
           },
         },
       },
+      "projectType": "application",
+      "root": "apps/foo",
+      "schematics": Object {
+        "@nrwl/angular:application": Object {
+          "e2eTestRunner": "cypress",
+          "linter": "eslint",
+          "unitTestRunner": "jest",
+        },
+        "@nrwl/angular:library": Object {
+          "linter": "tslint",
+        },
+      },
+    },
+  },
+  "schematics": Object {
+    "@nrwl/angular": Object {
+      "application": Object {
+        "linter": "tslint",
+      },
+      "library": Object {
+        "linter": "tslint",
+      },
     },
   },
   "version": 1,
```

Comment:
Wow, its weird seeing a snapshot that demonstrates the exact issue you were fixing. We should have caught this sooner 😦

Reply:
Yeah, the thing is all our tests use the `createTreeWithEmptyWorkspace` with no parameter and that creates the workspace with version 1. We might need to be mindful moving forward when we write tests to make sure it works well across the different versions.

Reply:
But I agree, this particular test case was overlooked. It was starting as a v1 workspace but it ended up as a v2 workspace and we didn't notice before.